tests/xharness/AppRunner.cs

```diff
@@ -331,6 +336,8 @@ public async Task<int> RunAsync ()
 			if (!isSimulator)
 				args.Add (new DisableMemoryLimitsArgument ());
 
+			// needed if a tunnel is used to connect to the device, but we cannot close it inside the else clause,
```

Comment:
I don't understand this comment. What is needed?

Reply:
To close it, I'll rephrase it.